src/runtime/workspace_pool.cc

```diff
@@ -124,6 +124,7 @@ WorkspacePool::WorkspacePool(
     : device_type_(device_type), device_(device) {}
 
 WorkspacePool::~WorkspacePool() {
+  // TODO: This problem should be fixed, or there will be memory leak.
```

Comment:
What does this TODO mean? Should we remove the follow code entirely or uncomment those code?

Reply:
As the comment says, there will be "Segmentation fault" with the following code, but, without the following code, there will be memory leak. So, I think uncommenting these code is a temporary solution, we may need to uncomment these code and fix the Segmentation fault.

Reply:
"TODO" has replaced with a comment